lib/tasks/publishing_api.rake

```diff
@@ -241,9 +241,14 @@ namespace :publishing_api do
   end
 
   desc "Republish all HTML attachments to the Publishing API"
-  task republish_html_attachments: :environment do
-    HtmlAttachment.where(attachable_type: "Edition", attachable_id: Edition.publicly_visible.select("id")).each do |a|
-      Whitehall::PublishingApi.bulk_republish_async(a)
+  task :republish_html_attachments, [:state] => :environment do |_, args|
+    args.with_defaults(state: "withdrawn")
```

Comment:
Oh cool, I didn't know you could do this - looks super handy.

Should there be a default state for this task though? the task wouldn't imply that you were dealing with withdrawn as the default.